source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -173,92 +189,125 @@ that matches your query, use the filtered positional operator
 The filtered positional operator ``$[<identifier>]`` specifies the
 matching array elements in the update document. To identify which array
 elements to match, pair this operator with ``<identifier>`` in an
-``options.arrayFilters`` object.
-
-The ``<identifier>`` term is a placeholder value you assign that represents
-an element of the array field name that prefixes it. This value must
-start with a lowercase letter and contain only alphanumeric characters.
+``arrayFilters`` object.
+
+The ``<identifier>`` value is a placeholder you assign that represents
+an element of the array field. This value must start with a lowercase
+letter and contain only alphanumeric characters.
+
+Usage
+`````
+
+Include a query, an update document, and
+optionally, an options object as parameters in an update operation.
+
+The following steps describe how to use a filtered positional operator
+in an update operation:
+
+1. Format your update document as follows:
+   
+   .. code-block:: javascript
+   
+      { $<operator>: { "<array>.$[<identifier>].<arrayField>": <updateParameter> } }
+
+   This update document contains the following placeholders:
+   
+   - ``$<operator>``: The array update operator
+   - ``<array>``: The array in the document to update
+   - ``<identifier>``: The identifier for the filtered positional operator
+   - ``<arrayField>``: The field in the ``<array>`` array element to update
+   - ``<updateParameter>``: The value that describes the update
+
+#. Add the matching criteria in the ``arrayFilters`` object. This object
+   is an array of queries that specify which array elements to include
+   in the update. Set this object in an ``options`` parameter:
+
+   .. code-block:: javascript
+      
+      arrayFilters: [
+        { "<identifier>.<arrayField1>": <updateParameter1> },
+        { "<identifier>.<arrayField2>": <updateParameter2> },
+        ...
+      ]
+
+#. Pass the query document, the update document, and options to an
```

Comment:
Suggestion:

I noticed it was referred to as "query" rather than "query document" in the first sentence of this section, so I think it would be good to stay consistent. Since the variable name in the code example is "query" rather than "queryDocument", maybe "query" would be appropriate.

```suggestion
#. Pass the query, the update document, and options to an
```